tests/Oryx.Tests.Common/TelemetryClientHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Oryx.Tests.Common;
+using System;
+
+public static class TelemetryClientHelper
+{
+    public static string aiKey = TestConstants.AiKey;
```

Comment:
nit: I'd personally remove this `aiKey` private variable since we can directly access` TestConstants.AiKey`